b2sdk/v2/bucket.py

```diff
@@ -21,6 +28,68 @@ def get_fresh_state(self) -> 'Bucket':
         except v3BucketIdNotFound as e:
             raise BucketIdNotFound(e.bucket_id)
 
+    @_file_infos_rename
+    def upload_bytes(
+        self,
+        data_bytes,
+        file_name,
+        content_type=None,
+        file_info: dict | None = None,
+        progress_listener=None,
+        encryption: v3.EncryptionSetting | None = None,
+        file_retention: v3.FileRetentionSetting | None = None,
+        legal_hold: v3.LegalHold | None = None,
+        large_file_sha1: Sha1HexDigest | None = None,
+        custom_upload_timestamp: int | None = None,
+        cache_control: str | None = None,
+    ):
+        return super().upload_bytes(
+            data_bytes=data_bytes,
+            file_name=file_name,
+            content_type=content_type,
+            file_info=file_info,
+            progress_listener=progress_listener,
+            encryption=encryption,
+            file_retention=file_retention,
+            legal_hold=legal_hold,
+            large_file_sha1=large_file_sha1,
+            custom_upload_timestamp=custom_upload_timestamp,
+            cache_control=cache_control,
+        )
```

Comment:
```suggestion
    @_file_infos_rename
    def upload_bytes(
        self,
        data_bytes,
        file_name,
        content_type=None,
        file_info: dict | None = None,
        progress_listener=None,
        encryption: v3.EncryptionSetting | None = None,
        file_retention: v3.FileRetentionSetting | None = None,
        legal_hold: v3.LegalHold | None = None,
        large_file_sha1: Sha1HexDigest | None = None,
        custom_upload_timestamp: int | None = None,
        cache_control: str | None = None,
        *args,
        **kwargs
    ):
        return super().upload_bytes(
            *args,
            data_bytes=data_bytes,
            file_name=file_name,
            content_type=content_type,
            file_info=file_info,
            progress_listener=progress_listener,
            encryption=encryption,
            file_retention=file_retention,
            legal_hold=legal_hold,
            large_file_sha1=large_file_sha1,
            custom_upload_timestamp=custom_upload_timestamp,
            cache_control=cache_control,
            **kwargs
        )
```
Future-proofing - if someone adds an argument, we want it to be visible in v2 too and we do not want to rely on someone remembering it should be added here too.